src/sentry/integrations/jira/webhooks.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+
+from django.views.decorators.csrf import csrf_exempt
+
+from sentry.api.base import Endpoint
+
+from sentry.integrations.atlassian_connect import AtlassianConnectValidationError, get_integration_from_jwt
+from sentry.models import sync_group_assignee_inbound
+
+
+class JiraIssueUpdatedWebhook(Endpoint):
+    authentication_classes = ()
+    permission_classes = ()
+
+    @csrf_exempt
+    def dispatch(self, request, *args, **kwargs):
+        return super(JiraIssueUpdatedWebhook, self).dispatch(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        try:
+            token = request.META['HTTP_AUTHORIZATION'].split(' ')[1]
```

Comment:
It's also possible that you'd raise a `KeyError` here if there was no Authorization header at all.

Can just avoid it by either `except (KeyError, IndexError)` or `request.META.get('HTTP_AUTHORIZATION', '').split(' ')[1]`

Also ProTip:

```python
>>> 'foo bar baz'.split(' ')
['foo', 'bar', 'baz']
>>> 'foo bar baz'.split(' ', 1)
['foo', 'bar baz']
```

You probably want the latter here, `.split(' ', 1)`